.github/workflows/backup-db.yml

```diff
@@ -30,6 +30,10 @@ jobs:
         with:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
 
+      - uses: DFE-Digital/github-actions/set-kubelogin-environment@master
```

Comment:
kubelogin is used by get-cluster-credentials so it must happen before it

Reply:
That's already above get-credentials? I just removed the duplicate before it